content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
- - /developers/metrics/gauges/
- - /developers/metrics/histograms/
- - /developers/metrics/rates/
- - /developers/metrics/sets/
- - /developers/metrics_type/
- - /developers/metrics/metrics_type/
+    - /developers/metrics/counts/
+    - /developers/metrics/distributions/
+    - /developers/metrics/gauges/
+    - /developers/metrics/histograms/
+    - /developers/metrics/rates/
+    - /developers/metrics/sets/
+    - /developers/metrics_type/
+    - /developers/metrics/metrics_type/
 further_reading:
-- link: "developers/dogstatsd"
-  tag: "Documentation"
-  text: "Learn more about DogStatsD"
-- link: "developers/libraries"
-  tag: "Documentation"
-  text: "Official and Community created API and DogStatsD client libraries"
+    - link: 'developers/dogstatsd'
+      tag: 'Documentation'
+      text: 'Learn more about DogStatsD'
+    - link: 'developers/libraries'
+      tag: 'Documentation'
+      text: 'Official and Community created API and DogStatsD client libraries'
 ---
 
-A metric's type affects how the given metric is aggregated in queries and values displayed in graph visualizations within Datadog. Different metric types can have different default time aggregation functions and different [Metric Type modifiers][1]. A metric's type is displayed on the details sidepanel for the given metric on the [Metrics Summary page][2]. Note: Changing the metric type in this details sidepanel can potentially change metric behavior in all existing visualizations and monitors; potentially rendering historical data as nonsensical.
+## Overview
+
+Each metric submitted to Datadog should have a metric type. A metric’s type affects how the metric values are displayed when queried as well as the associated graphing possibilities within Datadog using additional [modifiers][1] and [functions][2]. A metric’s type is displayed on the details side panel for the given metric on the [Metrics Summary page][3].
+
+**Note**: Changing the metric type in this details side panel can potentially change metric behavior in all existing visualizations and monitors; potentially rendering historical data as nonsensical.
 
 The following metric submission types are accepted:
 
-* [COUNT](?tab=count#metric-type-definition)
-* [RATE](?tab=rate#metric-type-definition)
-* [GAUGE](?tab=gauge#metric-type-definition)
-* [HISTOGRAM](?tab=histogram#metric-type-definition)
-* [DISTRIBUTION](?tab=distribution#metric-type-definition)
-* [SET](?tab=set#metric-type-definition)
-* [TIMERS](?tab=timers#metric-type-definition)
+- [COUNT](?tab=count#metric-types)
+- [RATE](?tab=rate#metric-types)
+- [GAUGE](?tab=gauge#metric-types)
+- [HISTOGRAM](?tab=hisogram#metric-types)
+- [DISTRIBUTION](?tab=distribution#metric-types)
+
+These different metric submission types are mapped to 4 in-app metric types found within the Datadog web application:
 
-Once a metric is submitted, there are 3 metric storage types within the Datadog web application:
+- COUNT
+- RATE
+- GAUGE
+- DISTRIBUTION
 
```

Comment:
Add in: 
"Note:  If you submit a metric to Datadog without a Metric Type, it will appear in-app as "Other". "Other" type cannot be further changed to another in-app type until an initial metric type is submitted.